src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -192,11 +190,11 @@ String layoutString(Type type, long align) {
         return switch (type) {
             case Primitive p -> primitiveLayoutString(p, align);
             case Declared d when Utils.isEnum(d) -> layoutString(((Constant)d.tree().members().get(0)).type(), align);
-            case Declared d when Utils.isStructOrUnion(d) -> alignIfNeeded(STR."\{JavaName.getFullNameOrThrow(d.tree())}.layout()", ClangAlignOf.getOrThrow(d.tree()) / 8, align);
-            case Delegated d when d.kind() == Delegated.Kind.POINTER -> alignIfNeeded(STR."\{runtimeHelperName()}.C_POINTER", 8, align);
+            case Declared d when Utils.isStructOrUnion(d) -> alignIfNeeded(String.format("%s.layout()", JavaName.getFullNameOrThrow(d.tree())), ClangAlignOf.getOrThrow(d.tree()) / 8, align);
```

Comment:
Same thing here - if all calls to `alignIfNeeded` end up with `String::format`, considering moving `String::format` there

Reply:
Fixed in [991ad7c](https://github.com/openjdk/jextract/pull/244/commits/991ad7c1a6f2c13aa9c7860cf654432e8bc70bac)